internal/distro/rhel9/imagetype.go

```diff
@@ -332,7 +332,7 @@ func (t *imageType) checkOptions(customizations *blueprint.Customizations, optio
 		}
 
 		if t.name == "edge-simplified-installer" {
-			allowed := []string{"InstallationDevice", "FDO", "Ignition", "Kernel"}
+			allowed := []string{"InstallationDevice", "FDO", "Ignition", "Kernel", "User"}
```

Comment:
note: This is the actual reason why the customization "started to work" for simplified-installer.